app/uk/gov/hmrc/helptosavefrontend/views/register/you_are_eligible.scala.html

```diff
@@ -47,17 +48,11 @@ <h1 class="govuk-heading-xl">
       <li>@Messages("hts.register.you-are-eligible.li2")</li>
     </ul>
 
-    @form(routes.EligibilityCheckController.youAreEligibleSubmit) {
-        @CSRF.formField(request)
-        @submit('_buttonClass -> "", 'id → "start-creating-account") {
-            @Messages("hts.global.continue-button")
-        }
-    }
```

Comment:
Why was this form removed?

Reply:
Now that we are using the govuk form, it's been replaced.